sumologic/resource_sumologic_cse_network_block.go

```diff
@@ -0,0 +1,126 @@
+package sumologic
+
+import (
+	"errors"
+	"github.com/hashicorp/terraform-plugin-sdk/helper/schema"
+	"log"
+)
+
+func resourceSumologicCSENetworkBlock() *schema.Resource {
+	return &schema.Resource{
+		Create: resourceSumologicCSENetworkBlockCreate,
+		Read:   resourceSumologicCSENetworkBlockRead,
+		Delete: resourceSumologicCSENetworkBlockDelete,
+		Update: resourceSumologicCSENetworkBlockUpdate,
+		Importer: &schema.ResourceImporter{
+			State: schema.ImportStatePassthrough,
+		},
+
+		Schema: map[string]*schema.Schema{
+			"address_block": {
+				Type:     schema.TypeString,
+				Required: true,
+				ForceNew: false,
+			},
+			"label": {
+				Type:     schema.TypeString,
+				Required: true,
```

Comment:
`label`, `internal`, and `suppresses_signals` are all optional in the API, why do we have them as required here?

Reply:
done under https://github.com/SumoLogic/terraform-provider-sumologic/pull/271/commits/4c00657834f23744c79da15a651cf03f35b3f5dc